sdk/keyvault/azure-keyvault-certificates/azure/keyvault/certificates/_parse_id.py

```diff
@@ -0,0 +1,41 @@
+# ------------------------------------
+# Copyright (c) Microsoft Corporation.
+# Licensed under the MIT License.
+# ------------------------------------
+
+from ._shared import parse_key_vault_identifier, ParsedId
+
+
+def parse_certificate_id(original_id):
+    # type: (str) -> ParsedId
+    """Parses a full certificate's ID into a class.
+
+    :param str original_id: the full original identifier of a certificate
```

Comment:
Let's settle on `source_id` for now, per JS. We're only in preview so we can change it later, but JS has had a head start in the discussion so at least we can be somewhat consistent for preview 1.